worldedit-sponge/src/main/java/com/sk89q/worldedit/sponge/SpongeWorldEdit.java

```diff
@@ -59,7 +58,9 @@
 /**
  * The Sponge implementation of WorldEdit.
  */
-@Plugin(id = SpongeWorldEdit.MOD_ID, name = "WorldEdit", version = "%VERSION%")
+@Plugin(id = SpongeWorldEdit.MOD_ID, name = "WorldEdit",
```

Comment:
Well you still need the `@Plugin` annotation with at least the plugin ID. The URL and description could be also defined in the Gradle file but considering it doesn't change every build we can also put it here.